### DISCUSSION TITLE
Unsupported item key.

### DISCUSSION THREAD

**Discussion by wjterveld** [selivan/https-ssl-cert-check-zabbix#41] [General]
Hi,
I wanted to use the advance version and integrate is with Zabbix..
But when I do the check I get "ssl_cert_check_   [m|ZBX_NOTSUPPORTED] [Unsupported item key.]"
It seems there is something missing? Could it be that it cannot find the "ssl_cert_list" file in the "/etc/zabbix/scripts" folder? I had to create this script folder so maybe that is why?

Zabbix version is: 5.2.1

Willem Jan

**Comment by wjterveld** [comment]
![image](https://user-images.githubusercontent.com/4067262/207590363-1f781fe5-0097-4cc9-8303-7c6f04733d5d.png)

**Comment by dsm-team** [comment]
I have the same issue.
Trying to figure out where to place the file which contains the host list
I suppose /etc/zabbix/scripts is a folder which exist in the specific installation of Selivan.
I have tried to put it in my externalscript folder with no luck and the zabbix documentation doesn't give much information
Couldn't find how this works regarding the "key" parameter in the context of this autodiscovery

**Comment by wjterveld** [comment]
runnig zabbix_server --help returns some info like..
  ExternalScripts                "/usr/lib/zabbix/externalscripts"
But putting the script and the file with the hosts there does not make it work!

**Comment by keys4dev** [comment]
I had this issue too.

1 - Put userparameters_ssl_cert_check.conf in "/etc/zabbix/zabbix_agentd.d"

2- Put ssl_cert_check.sh script is in same folder that the path in the file  "userparameters_ssl_cert_check.conf" (see screenshot red line).

![image](https://user-images.githubusercontent.com/75089820/210839057-daf4728c-1f1b-41a6-b1e3-a7cceadedfb9.png)

3-Ensure you can execute the script:
chmod +x ssl_cert_check.sh

4- Restart your zabbix instance:
sudo systemctl reload zabbix-server

**Reply by MohamadJavan** [reply]
i did this but problem exist
